src/goto-programs/goto_contractor.cpp

```diff
@@ -164,6 +182,8 @@ void goto_contractort::contractor()
 
   oss << "\n\t- Domains (after): " << X;
   map.update_intervals(X);
+  if(X.is_empty() || X[0].is_degenerated())
```

Comment:
Please add a comment.

Reply:
done. This is to check if interval box is empty set or if the interval is degenerated in the case of a single interval.